### DISCUSSION TITLE
thermal conductivity from different versions

### DISCUSSION THREAD

**Discussion by mayankaditya** [deepmodeling/deepmd-kit#1457] [Q&A]
Dear Developers,
I have computed the thermal conductivity using the GK method, with force-field generated from deepmd-1.3.3; the results nicely agree with the experiments; they also show converged values. When I retrained the model using the 2.0.2 version,  the results were entirely different (now showing large fluctuations in HACF and KAPPA data) with the same supercell size. Other features like PDF and MSD agree well between different versions.

I found this issue with many other systems as well.
Kindly suggest a possible solution. 

Thanks,
Mayank

**Comment by wanghan-iapcm** [comment]
Dear Mayank, 

The implementation of heat conductivity in 1.3.3 is actually not entirely correct. We fix the issue in #1093 . Please find more details there. 

Best,
Han

**Comment by amcadmus** [comment]
I convert this issue to discussion.
